app/models/bundle_context.rb

```diff
@@ -22,13 +22,21 @@ class BundleContext < ApplicationRecord
   }
 
   enum content_metadata_creation: {
-    "default_style" => 0,
-    "filename_style" => 1,
-    "smpl_style" => 2
-  }
+    "default_cm_style" => 0,
+    "filename_cm_style" => 1,
+    "smpl_cm_style" => 2
+    }
+
+  def content_md_creation
+    content_metadata_creation
+  end
+
+  def project_style
+    content_structure
+  end
 
   def staging_dir
-    '/dor/assembly'
+    '/tmp/assembly'
```

Comment:
in production, this will be /dor/assembly ... this needs to be a setting.  Fine to ticket it separately.

Reply:
#273